tests/acceptance/features/bootstrap/WebUIUsersContext.php

```diff
@@ -627,4 +627,32 @@ public function theAdministratorChangesTheEmailOfUserToUsingTheWebui($username,
 		$this->usersPage->setSetting('Show email address');
 		$this->usersPage->changeUserEmail($this->getSession(), $username, $email);
 	}
+
+	/**
+	 * @Then /^the user count of group "([^"]*)" should display (\d+) users in the webUI$/
```

Comment:
we standardized to `on the webUI`, so let's try to keep that.
```suggestion
	 * @Then /^the user count of group "([^"]*)" should display (\d+) users on the webUI$/
```